configs/sites/frontera/packages.yaml

```diff
@@ -0,0 +1,256 @@
+packages:
+  all:
+    compiler:: [intel@23.1.0]
+    #compiler:: [gcc@12.2.0]
+    providers:
+      mpi:: [intel-oneapi-mpi@2021.9.0]
+      #mpi:: [mvapich2@2.3]
```

Comment:
Either use a version of openmpi or whatever MPI you want to use with a gnu-9.1.0 stack, or delete the line

Reply:
Done